Sources/Controllers/MessagesViewController.swift

```diff
@@ -49,6 +57,9 @@ open class MessagesViewController: UIViewController {
     open override var shouldAutorotate: Bool {
         return false
     }
+    
+    /// A Boolean value used to determine if `viewDidLayoutSubviews()` has been called.
```

Comment:
It will be false if `viewDidLayoutSubviews()` has been called, so maybe should it be "has not been called"